EIPS/eip-7532.md

```diff
@@ -0,0 +1,44 @@
+---
+eip: 7532
+title: Generation unit (64 epochs)
+description: A Generation is a scale version of an epoch.
+author: JKincorperated (@JKincorperated)
+discussions-to: https://ethereum-magicians.org/t/another-eip-a-new-unit-the-generation/16024
+status: Draft
+type: Informational
```

Comment:
We don't generally make EIPs solely to define a term. Instead terms are normally defined within a technical specification using that term.

We don't really have a process to force people to use terms correctly, so other documents would be free to define Generation however they like.